packages/plugin-coinbase/src/utils.ts

```diff
@@ -423,7 +423,7 @@ export async function executeTransferAndCharityTransfer(
     const assetIdLowercase = sourceAsset.toLowerCase();
 
     let charityTransfer: Transfer;
-    if (charityAddress && charityAmount > 0) {
+    if (false) {
```

Comment:
Why is that? If we want to keep that code for later refactor, let's leave TODO comment

Reply:
When transfers are too small it fails so removing for now will add it back in this weekend don't worry :)